### ISSUE TITLE
Accessibility - Aria-label is missing from Search box

### ISSUE THREAD

**Issue by MariaAug** [microsoft-search/pnp-modern-search#1337]
**Version used**
4.2.2.0

**Describe the bug**
As aria-label is missing on the search box, only if the voice command is provided as click 'Search field' it works using Dragon tool.

**To Reproduce**
1. Add PnP search box web part on a SharePoint page.
2. Save the page.
3. Provide voice command as click Search news - This will not work
4. Provide voice command as click Search field - This will work

**Expected behavior**
Aria-label should be same as placeholder for the user to see and dictate the command.

**Screenshots**
![image](https://user-images.githubusercontent.com/90712499/134520468-b02a7939-127d-4075-b282-1be4796f992d.png)
